src/functions/post.php

```diff
@@ -0,0 +1,133 @@
+<?php
+if ( ! function_exists( 'tribe_post_count' ) ) {
+	/**
+	 * Counts the amount of posts we have for a given Series, it's important to note this will be cached
```

Comment:
After reading through this function, it looks like you can genericize _Series_ down to _post_type_.